tests/e2e/commands/network.go

```diff
@@ -81,11 +81,30 @@ func StartNetworkWithVersion(version string) string {
 }
 
 /* #nosec G204 */
-func StopNetwork() {
+func StopNetwork(stopCmdFlags ...string) {
```

Comment:
A good way to avoid having many functions for the different commands, is to have just one function
ExecuteCLICommand or something like that, which receives the cmd name and the flags
(it may only add the skip update flag)

Reply:
anyway I recognize the e2e code will be easier to read if we have one function per command

Reply:
in that later case make all functions call this generic one

Reply:
we are calling `exec.Command` - is that what you are mentioning here?

Reply:
I mean, all the body inside here can be abstracted away in a separate function, with arguments:
- NetworkCmd
- "stop"
- stopCmdFlags

Reply:
let me see how other NetworkCmd e2e tests look in my next PRs. will do this should I see this pattern in more tests